SIMPLE/.classpath

```diff
@@ -15,5 +15,6 @@
 	<classpathentry kind="lib" path="lib/commons-io-2.11.0.jar"/>
 	<classpathentry kind="lib" path="lib/icepdf-core-6.2.2.jar"/>
 	<classpathentry kind="lib" path="lib/icepdf-viewer-6.2.2.jar"/>
+	<classpathentry kind="lib" path="C:/Users/sameh/Documents/GitHub/Route/simple-java/SIMPLE/lib/miglayout-4.0.jar"/>
```

Comment:
I think you used "add external jar" in Eclipse. This classpathentry won't work on our machines. Can you change it to the ones listed in other lines?